pages/views/_includes/landing/blogs.njk

```diff
@@ -0,0 +1,16 @@
+<div class="blog container">
+  <h2>{{ landing.blogs.title }}</h2>
+  <esl-animate target="::next::child" group></esl-animate>
+  <ul class="blog-container">
+    {% for item in landing.blogs.items | sortByDate | limit(3) %}
```

Comment:
We should use the pages of the blog (from the collection) instead of providing them manually in the landing content